bin/rbenv-doctor

```diff
@@ -18,6 +18,10 @@ indent() {
   sed 's/^/  /'
 }
 
+inode-id() {
+  xargs stat -c '%i'
```

Comment:
I'm worried about the portability of this solution. For example, `stat -c` doesn't work for me on macOS.

I would suggest that we simplify the approach. How about ignoring the cases when rbenv was found in both `/bin` and `/usr/bin`? That would solve our Ubuntu problem and avoid having to know anything about symlinks.

Reply:
Ahh, that sucks.

Reply:
I changed it but it's not as succinct now :P

Reply:
You can drop this in the file to do a simple test of scenarios.

```
echo "====== Poor man unit test ====="
echo "Should be 1 (only one): $(echo -e "/usr/bin/rbenv" | remove-bin-symlink | uniq | count-lines)"
echo "Should be 1 (remove /bin): $(echo -e "/usr/bin/rbenv\n/bin/rbenv" | remove-bin-symlink | uniq | count-lines)"
echo "Should be 2 (custom install): $(echo -e "/usr/bin/rbenv\n/home/.rbenv/bin/rbenv" | remove-bin-symlink | uniq | count-lines)"
echo "==============================="
```

Reply:
Thanks! You don't have to force-push every time; I usually squash multiple commits into one when merging. 👍